content/docs/1.6.0/references/settings.md

```diff
@@ -470,7 +470,7 @@ Number of millicpus on each node to be reserved for each instance manager pod wh
 
 > **Warning:**
 >  - Specifying a value of 0 disables CPU requests for instance manager pods. You must specify an integer between 1000 and 8000. 
->  - This is a global setting. Modifying the value triggers an automatic restart of the instance manager pods. Do not modify the value while volumes are still attached.
+>  - This is a global setting. Modifying the value triggers an automatic restart of the instance manager pods. After this setting is changed, the v2 instance manager pod using this global setting will be automatically restarted without instances running on the v2 instance manager.
```

Comment:
> v2 instance manager pod using this global setting will be automatically restarted without instances running on the v2 instance manager.

This is unclear. Do you mean that the pod is restarted only when no instances are running on the Instance Manager?

Reply:
Yes, it is.